pywr/notebook/__init__.py

```diff
@@ -212,50 +313,3 @@ def create_node_class_trees():
         clss = [cls.__name__.lower() for cls in classes[::-1]]
         node_class_trees[name] = clss
     return node_class_trees
-
-
-def draw_graph(model, width=500, height=400, labels=False, attributes=False, css=None):
```

Comment:
We could replace this with a function that creates a schematic instance and calls `draw_graph`?

Reply:
Done